042-SAPOnAzure/Student/06-Start-Stop-Automation.md

```diff
@@ -0,0 +1,459 @@
+
+![image](https://user-images.githubusercontent.com/26795040/115051125-19d1c300-9ea2-11eb-9df9-e2d5425e3c70.png)
+
+Information about SAP landscape and instances are to be stored in VM Tags.
+
+Secure assets in Azure Automation include credentials, certificates, connections, and encrypted variables. These assets are encrypted and stored in Azure Automation using a unique key that is generated for each Automation account. [Azure Automation stores the key in the system-managed Key Vault](https://docs.microsoft.com/en-us/azure/automation/shared-resources/credentials?tabs=azure-powershell). Before storing a secure asset, Automation loads the key from Key Vault and then uses it to encrypt the asset.
+
+SAP system start / stop / monitoring and SAP Application server Start / stop is implemented using scripts (calling SAP sapcontrol executable) via the Azure VM agent. 
+
+SAP HANA start / stop / monitoring is implemented using scripts (calling SAP sapcontrol executable) via the Azure VM agent. 
+
+SQL Server start / stop / monitoring is implemented using scripts (calling SAP Host Agent executable) via the Azure VM agent. 
+
+Azure runbooks can either be executed manually or scheduled. 
+
+## Success Criteria
+
+- Create Azure Automation account
+- Tag all the VMs for the **SID** in place
+- List the SID to ensure all the systems were tagged correctly
+- Execute runbook manually to stop systems
+- Schedule the runbook to start the systems
+- (Optional) Restrict access to runbooks to an individual user
+
+## Learning Resources 
+
+- [Manage credentials in Azure Automation](https://docs.microsoft.com/en-us/azure/automation/shared-resources/credentials?tabs=azure-powershell)
+ 
+- [Azure Quickstart - Create an Azure Automation account](https://docs.microsoft.com/en-us/azure/automation/automation-quickstart-create-account)
+
+- [SAP Library - Tools for Monitoring the System](https://help.sap.com/saphelp_aii710/helpdata/en/f7/cb8577bc244a25a994fc3f9c16ce66/frameset.htm)
+
+- [Azure Automation Start/Stop VMs during off-hours overview](https://docs.microsoft.com/en-us/azure/automation/automation-solution-vm-management)
+
+- [Pricing - Managed Disks](https://azure.microsoft.com/en-us/pricing/details/managed-disks/)
+
+- [Pricing - Automation](https://azure.microsoft.com/en-us/pricing/details/automation/)
+
+
+### **Import Az.Modules**
+
+SAP start / stop PowerShell (PS) runbooks use new Az PS module, so import following AZ modules:
+
+  - **Az.Account**
+
+  - **Az.Compute**
+
+  - **Az.Automation**
+
+  - **Az.Resources**
+
+
+### **Import SAP PowerShell Module**
+
+Import **SAPAzurePowerShellModules** PowerShell module that will be used by SAP Runbooks.
+
+> [!NOTE]
+> PowerShell module **SAPAzurePowerShellModules** is stored in PowerShell Gallery and is easy to import into Azure automation account.
```

Comment:
Please don't use the greater-than sign. This creates a "quoted block" and we don't use those in WTH ever.

You should put this on one line, starting with "NOTE:" in bold and then use the backticks to make "**SAPAzurePowerShellModules** monospace font (and don't make it bold).

Reply:
fixed.